### PR TITLE
Feature 69 font awesome qpm test

### PR DESCRIPTION
**Pull request by romansavelyev**
test for appVeyor

### CODE REVIEW

File: doc/workflow/qpm_packages_installation.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+To work with PPS 2.0 you need to install Font Awesome web fonts package manager, so:
+-  download [qpm package manager](https://www.qpm.io/). 
+-  put qpm.exe in PPS project folder.
+-  use `qpm install com.cutehacks.fontawesome` in command line.
```
Comment (Guitarheroua) [inline doc/workflow/qpm_packages_installation.md]: Please, change `qpm install com.cutehacks.fontawesome` to `qpm install`

File: doc/workflow/qpm_packages_installation.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+-  `include($$PWD/vendor/vendor.pri)` in pllug-presentation-system.pro file.
```
Comment (Guitarheroua) [inline doc/workflow/qpm_packages_installation.md:5]: You need to remove information, started from there, as we already added this code to our project. So people do not need to add it by them self.

File: doc/workflow/qpm_packages_installation.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+-  add in main.cpp file:       
+      ```
+      QQmlApplicationEngine engine;
+      engine.addImportPath(QStringLiteral("qrc:/"));
+      engine.load(QUrl(QStringLiteral("qrc:/main.qml")));
+      ```
```
Comment (Guitarheroua) [inline doc/workflow/qpm_packages_installation.md]: Please, add information about usage of FontAwesome in QML files(write about `import com.cutehacks.fontawesome 1.0`)

**Comment by Guitarheroua** [comment]
Do you change README.md files, as we discussed? Because I did not find any changes there!

**Comment by romansavelyev** [comment]
@Guitarheroua  'import com.cutehacks.fontawesome 1.0' is already in the project, should i remove it and add info about it in qpm_packages_installation.md ?

**Comment by romansavelyev** [comment]
@Guitarheroua if i change `qpm install com.cutehacks.fontawesome` to `qpm install` than `qml.json` will not appear in the project folder and so do `vendor` folder

**Comment by Guitarheroua** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/PLLUG/pllug-presentation-system-2/pull-request/84/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=PLLUG/pllug-presentation-system-2)

**Pull request merged**